app/build.gradle

```diff
@@ -150,6 +150,9 @@ fladle {
 }
 
 dependencies {
+    implementation project(path: ':autofill-api')
+    implementation project(path: ':autofill-impl')
```

Comment:
We can add autofill-store here so we don't need to expose as the store as api in autofill-impl

Reply:
I don't fully understand that statement 😅

Do you want to make the changes and just push them (as the best way of illustrating your point)?

Reply:
The store should never be exposed. If you need something from the store it should be exposed via the api module.

Reply:
+1 to what @marcosholgado says. Only `:api` modules should be public interface. Anything else should be considered private and never access them directly.

Reply:
We discussed this on a call, and concluded:

- `implementation project(path: ':autofill-store')` is good to have in the app/build.gradle, preferred over use of `api` elsewhere (ie, we should always use `implementation` and never `api` to standardise the approach)

Reply:
>If you need something from the store it should be exposed via the api module.

We aren't knowingly accessing anything from `impl` or `store` from the browser.